src/main/java/net/dv8tion/jda/api/events/channel/update/ChannelUpdateArchiveTimestampEvent.java

```diff
@@ -19,15 +19,31 @@
 import net.dv8tion.jda.api.JDA;
 import net.dv8tion.jda.api.entities.channel.Channel;
 import net.dv8tion.jda.api.entities.channel.ChannelField;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
 import net.dv8tion.jda.internal.utils.Helpers;
 
 import javax.annotation.Nonnull;
 import java.time.OffsetDateTime;
 
-//TODO-v5: Docs
+/**
+ * Indicates that a {@link Channel} archival timestamp was updated.
+ *
+ * <p>This timestamp will be updated when any of the following happens:
+ * <ul>
+ *     <li>The channel is archived</li>
+ *     <li>The channel is unarchived</li>
+ *     <li>The AUTO_ARCHIVE_DURATION is changed.</li>
```

Comment:
This `.` either needs to be removed or the above two to add. I'm not sure which is the normal convention.